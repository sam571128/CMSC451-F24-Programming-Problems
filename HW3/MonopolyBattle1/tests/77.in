47 1498 355575956
8 37
36 33
8 10
11 12
30 8
42 38
47 29
19 37
6 11
46 13
16 23
24 19
18 31
19 28
16 20
37 13
44 26
24 6
42 25
28 6
9 45
14 24
45 41
29 21
42 28
14 4
46 21
9 9
29 46
21 40
16 11
36 21
2 42
44 37
43 46
17 14
41 9
40 10
11 6
16 9
42 4
1 4
13 25
21 25
43 28
3 13
36 24
8 29
23 40
30 1
19 43
16 34
21 24
17 31
39 23
5 44
10 17
13 44
6 36
42 30
29 4
43 8
17 42
4 24
9 36
19 8
7 37
11 46
42 23
19 42
37 8
30 40
9 2
45 13
14 11
42 36
41 24
46 7
45 47
13 18
45 24
12 9
31 16
23 29
34 6
17 43
37 17
45 26
17 4
33 42
44 40
38 28
39 31
17 39
3 33
43 15
23 21
37 10
39 3
27 43
15 13
5 12
28 10
8 44
25 39
35 47
17 37
15 40
27 25
1 39
43 2
19 2
30 30
44 18
26 9
9 41
8 38
33 37
9 4
7 4
10 10
4 47
36 10
26 46
36 26
18 8
23 10
44 23
28 24
40 14
46 31
25 14
2 3
29 35
13 43
12 23
16 44
5 23
19 45
17 45
35 14
33 18
47 3
18 42
45 6
13 9
35 11
24 37
43 14
12 14
30 46
22 34
7 35
31 36
31 20
22 35
11 16
28 5
5 1
23 43
20 6
22 39
38 37
20 12
28 28
7 18
8 30
18 32
36 37
1 47
45 27
15 26
38 15
28 37
42 14
44 28
28 31
16 32
44 32
33 9
47 37
13 33
18 5
42 42
33 36
13 14
24 40
35 12
9 20
47 5
14 5
12 12
24 38
47 44
15 15
43 22
24 44
5 8
34 25
2 44
40 20
32 39
7 16
11 14
36 3
16 4
47 23
36 36
17 18
28 43
17 30
14 10
10 38
44 9
10 11
11 42
32 23
21 26
47 6
4 8
11 40
28 30
33 47
19 20
37 25
22 25
33 2
6 9
38 34
3 11
17 47
12 16
46 19
40 34
4 20
45 29
11 45
11 18
35 20
38 21
22 2
1 37
39 24
47 27
25 15
15 42
31 43
26 16
42 8
12 27
3 37
27 21
3 38
4 31
5 11
35 37
7 13
20 16
37 6
32 21
2 27
22 20
30 37
17 13
18 38
38 16
23 20
39 18
18 23
40 9
41 35
28 2
40 44
16 33
26 21
39 21
24 29
44 12
13 1
4 36
47 15
46 3
42 10
12 32
21 2
31 1
33 40
27 22
37 5
7 14
13 31
41 30
19 22
40 37
34 26
30 45
1 27
21 15
19 4
27 45
42 3
30 5
43 29
34 18
41 29
41 21
10 23
24 15
4 23
27 46
8 7
28 8
32 27
6 12
3 41
34 15
27 31
47 11
7 10
5 32
21 36
36 28
13 22
6 19
35 45
34 16
38 35
6 14
33 27
13 23
32 11
21 44
32 29
43 33
5 47
12 44
16 1
18 13
25 36
21 47
32 26
32 47
1 36
37 35
32 14
4 9
44 4
14 8
28 1
4 35
27 44
21 34
38 17
42 17
45 4
16 35
11 19
22 32
33 31
24 13
33 33
38 11
42 20
9 33
16 39
33 26
33 32
3 1
13 4
4 14
44 17
5 35
37 40
19 24
29 18
47 12
29 13
29 28
21 12
11 25
28 9
27 2
45 8
35 40
28 40
31 8
14 38
36 42
44 14
44 47
43 36
42 12
33 17
37 26
34 8
43 32
26 29
37 31
30 38
11 31
26 39
2 30
9 32
33 43
30 35
32 45
44 3
1 28
29 29
42 37
1 46
3 20
3 32
29 14
30 10
34 40
34 21
32 17
44 30
1 29
5 40
28 47
17 16
46 14
10 26
14 28
24 8
2 2
40 1
21 32
14 22
15 9
6 30
46 9
14 18
27 39
47 9
31 35
40 7
28 38
29 24
35 13
43 42
37 1
40 29
39 11
11 39
20 18
24 39
36 25
38 43
9 16
41 38
36 46
3 2
17 11
14 17
13 6
32 9
7 38
46 36
23 15
41 13
18 45
37 3
16 41
32 42
28 36
16 13
41 41
20 5
30 26
44 44
16 37
42 5
21 11
44 33
34 42
9 18
20 32
23 35
34 7
17 8
37 39
7 20
38 31
21 39
1 21
30 39
9 22
19 39
30 3
19 9
10 25
33 15
13 30
1 14
40 47
41 28
2 32
19 3
32 28
8 14
47 30
34 36
14 9
33 20
44 35
20 41
27 32
46 39
38 13
32 20
32 44
47 31
18 10
26 23
22 14
10 5
25 25
18 46
10 16
38 22
8 20
9 26
46 27
28 34
19 47
39 22
38 3
35 19
19 18
3 21
24 21
34 32
20 36
6 4
7 34
45 15
45 40
13 40
44 21
27 5
19 16
31 47
22 15
25 11
36 23
2 47
7 40
38 44
8 8
10 34
24 4
39 6
30 47
7 39
27 30
26 24
38 27
23 7
35 22
46 25
30 24
3 44
21 20
11 35
15 4
43 25
9 46
26 45
15 8
17 15
46 37
42 39
28 26
41 2
45 37
44 22
14 37
21 19
26 1
46 47
18 40
5 20
38 26
36 18
15 18
1 12
15 7
1 42
15 44
28 27
4 42
6 8
41 43
3 12
21 30
14 12
13 28
15 47
26 47
38 8
40 38
28 16
24 34
33 8
16 45
35 5
29 32
38 20
4 21
25 38
42 45
32 30
32 32
40 40
26 41
12 35
31 3
8 1
2 14
20 37
32 41
38 42
35 17
6 18
37 32
16 25
38 29
35 29
34 29
43 1
24 26
12 31
47 1
16 2
30 21
8 47
2 11
41 3
5 27
14 14
39 32
43 35
37 12
39 33
34 31
14 41
47 7
3 6
35 8
28 45
25 33
43 21
43 26
19 35
34 20
11 17
8 27
19 36
33 19
27 42
28 19
4 12
40 5
36 19
35 35
4 32
36 34
17 24
17 2
29 16
29 41
11 10
44 24
5 5
1 11
36 9
38 23
9 15
19 30
17 33
41 15
26 25
18 30
45 2
28 44
47 2
15 29
26 34
17 44
21 29
18 17
46 15
26 6
12 5
39 39
38 19
25 22
31 4
31 12
46 1
19 19
7 15
18 37
10 37
16 47
34 44
19 7
22 23
44 15
10 35
8 24
27 4
40 19
22 31
40 28
40 4
31 28
26 11
12 3
30 22
14 25
24 16
4 13
40 13
43 34
47 19
46 26
5 30
27 20
23 30
14 3
38 47
30 13
13 47
40 22
31 39
20 43
12 19
26 31
14 27
15 38
7 9
12 1
47 20
20 40
31 46
10 19
26 30
44 5
34 12
21 16
20 38
2 4
29 9
36 11
11 26
39 19
25 31
40 39
47 10
18 19
10 32
17 9
45 34
1 6
7 21
45 22
5 41
7 1
8 33
33 11
23 2
35 2
31 17
3 18
31 42
9 11
15 12
2 10
6 38
1 22
27 7
38 10
11 24
32 31
13 16
21 23
43 45
17 36
36 47
21 14
36 40
2 29
13 21
44 45
40 8
13 35
29 27
30 42
46 41
2 5
25 47
27 37
30 6
38 14
22 4
37 15
39 20
39 17
27 40
22 11
42 35
40 41
34 38
2 28
8 9
4 26
32 1
30 16
8 11
16 7
27 27
35 23
12 26
14 30
35 24
34 46
47 41
35 28
15 14
22 45
22 40
7 43
32 7
6 13
25 21
10 18
17 25
21 41
23 38
23 5
6 3
46 11
2 17
1 33
34 11
4 43
37 9
18 1
38 6
19 12
20 2
15 2
27 38
47 43
1 34
1 3
33 13
2 26
33 44
42 26
31 6
27 10
44 25
33 14
37 18
6 27
47 16
25 13
21 33
11 36
18 41
35 15
32 37
21 43
45 10
20 21
4 16
25 35
29 12
18 44
8 42
43 12
24 42
39 10
45 30
37 2
37 45
34 43
33 34
1 26
16 3
35 38
47 18
42 46
14 42
35 4
39 9
20 11
31 18
11 27
45 11
6 10
36 15
47 46
47 17
3 25
46 23
1 16
33 7
42 47
6 39
40 24
14 20
43 44
42 27
10 4
5 9
23 33
31 34
8 35
7 31
32 24
35 30
1 45
7 36
46 18
46 38
38 30
33 3
45 19
37 30
43 40
28 4
15 17
23 31
45 36
40 33
36 2
47 36
14 33
9 44
33 39
41 8
14 45
26 27
8 46
27 13
24 35
25 3
34 22
15 43
46 24
38 1
5 15
43 24
19 17
24 12
41 12
46 2
25 4
45 18
39 36
35 27
41 23
25 41
9 38
23 36
23 37
21 37
14 15
21 9
31 31
45 33
6 32
32 19
12 2
3 16
32 43
14 23
17 29
15 36
41 6
45 44
6 29
25 10
5 37
35 9
11 21
12 10
2 24
18 4
28 23
34 13
15 5
6 22
10 7
12 24
35 10
12 25
36 14
38 36
32 35
37 20
34 39
43 9
14 47
27 15
20 7
37 22
4 17
40 3
6 2
18 34
3 42
43 39
34 9
42 7
2 21
46 45
32 2
34 45
28 17
37 19
23 14
47 40
33 35
32 10
46 4
30 28
23 9
34 37
26 10
6 47
3 3
35 21
29 11
19 10
36 32
7 44
24 25
27 16
8 32
4 30
21 35
43 13
41 42
9 47
30 34
36 22
16 24
26 28
46 34
41 46
27 1
28 35
39 4
5 31
36 6
24 23
21 22
45 46
15 24
21 21
4 39
9 14
42 6
16 12
40 32
35 34
20 28
2 39
4 37
15 31
27 17
16 6
17 7
5 21
15 21
16 21
20 33
17 19
25 19
19 23
30 43
40 12
18 21
18 11
4 41
1 35
26 44
13 17
32 16
20 10
39 46
18 24
33 25
45 12
11 47
22 44
15 6
38 2
11 8
6 33
39 47
31 24
29 25
15 23
34 23
20 35
36 7
31 9
37 21
8 39
5 25
13 10
10 15
40 26
1 31
20 26
22 21
39 41
8 41
35 3
43 17
37 28
35 7
23 23
23 44
16 43
24 32
5 13
34 4
46 28
9 12
9 34
27 33
29 19
24 43
3 46
40 18
47 22
30 18
43 41
3 15
2 41
23 6
16 8
29 15
4 7
14 43
33 10
7 42
40 2
3 29
6 6
12 33
35 33
12 13
35 39
22 18
3 30
17 27
24 10
31 44
43 4
1 23
4 10
25 40
6 41
15 46
23 8
3 14
28 22
9 10
5 45
37 38
41 25
32 3
5 24
30 14
13 37
28 3
44 6
17 6
6 25
12 46
15 10
44 31
19 21
35 32
13 26
34 33
27 36
13 45
7 41
2 46
10 9
45 23
21 3
28 33
44 13
4 1
39 44
16 40
7 33
2 20
22 42
45 16
27 23
25 16
2 40
42 15
32 25
30 31
27 11
13 12
5 14
11 13
38 32
3 8
37 29
16 27
36 39
19 46
11 38
15 1
3 7
36 5
31 11
21 1
41 19
13 34
39 7
20 24
35 6
2 43
5 33
10 43
7 29
14 32
18 28
11 33
41 47
22 24
27 3
25 27
11 37
22 13
30 19
22 17
24 9
6 23
39 40
13 46
36 16
41 16
39 12
39 30
36 4
3 24
24 14
1 24
12 37
23 26
46 12
11 2
12 40
24 33
29 26
42 31
31 29
29 38
31 25
5 19
2 37
23 46
15 41
11 29
28 14
18 29
9 23
30 20
21 10
23 22
26 5
30 41
22 36
21 38
34 28
14 1
16 30
3 17
16 5
19 14
43 3
31 38
42 43
19 1
9 8
23 24
2 9
25 37
39 13
18 9
1 10
42 34
42 19
18 12
5 4
17 1
4 11
13 13
41 22
39 45
26 42
37 42
21 45
22 16
3 35
1 30
4 4
21 8
2 35
17 26
16 42
39 42
42 11
19 38
45 28
31 30
26 7
13 32
26 26
36 38
24 45
17 32
37 27
20 22
30 29
5 39
3 26
42 24
12 34
14 13
11 5
37 36
32 4
18 16
24 47
14 39
46 20
24 18
7 28
13 8
39 35
23 28
29 33
35 41
24 5
44 16
17 28
19 29
16 36
37 11
25 28
43 16
45 17
14 29
7 23
17 34
32 12
42 32
47 21
26 18
4 29
26 36
22 43
16 14
43 27
29 10
18 33
23 4
8 13
18 26
9 25
32 36
23 12
33 38
29 5
36 35
4 18
44 34
16 15
29 31
12 21
30 32
8 17
9 19
25 34
43 23
46 8
23 25
41 31
12 11
29 47
46 46
46 42
38 46
37 46
18 22
33 1
46 22
27 41
43 5
21 4
41 44
26 13
34 10
47 28
41 4
42 22
10 36
22 6
12 45
31 40
41 10
45 21
34 24
38 5
44 46
43 7
17 20
2 13
24 46
32 33
13 11
20 47
36 1
23 42
19 27
7 45
35 16
6 21
40 31
40 23
9 35
12 39
35 43
38 39
24 27
4 45
13 38
31 10
38 41
42 33
10 14
27 35
11 15
28 11
31 22
7 27
27 14
27 19
5 34
29 17
23 17
12 41
9 3
42 29
24 24
29 7
14 31
44 10
8 45
1 40
4 46
28 7
10 47
13 2
28 46
35 25
7 7
16 19
43 37
43 31
31 2
2 34
42 1
12 30
27 12
9 42
45 42
25 6
33 21
