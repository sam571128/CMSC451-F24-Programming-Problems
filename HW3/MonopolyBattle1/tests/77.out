64 82 99 64 158 64 78 64 82 64 64 64 64 64 96 83 64 80 64 83 64 196 69 64 64 64 171 64 64 64 64 168 64 83 134 64 64 64 94 64 82 64 64 64 82 64 82 